plugins/org.jkiss.dbeaver.model.jdbc/src/org/jkiss/dbeaver/model/impl/jdbc/data/JDBCCollection.java

```diff
@@ -192,9 +192,8 @@ public Array getArrayValue() throws DBCException {
         }
         final DBSDataType dataType = getComponentType();
         try (DBCSession session = DBUtils.openUtilSession(new VoidProgressMonitor(), dataType, "Create JDBC array")) {
-            if (session instanceof Connection) {
-                String typeName = dataType.getFullTypeName();
-                return ((Connection) session).createArrayOf(typeName, attrs);
+            if (session instanceof Connection connection) {
+                return connection.createArrayOf(dataType.getTypeName(), attrs);
```

Comment:
Are we sure that it won't break it in PG, Oracle and other databases?

Reply:
We are not, so those databases must be tested as well